examples/preprocessors__grep/README.md

```diff
@@ -0,0 +1,69 @@
+# Selecting tests using Mocha-like "fgrep" and "grep"
+
+This example uses [cypress-select-tests](https://github.com/bahmutov/cypress-select-tests) preprocessor plugin to filter specs and tests. It is similar to how [Mocha](https://mochajs.org/) has `--fgrep` and `--grep` CLI arguments. This project provides imitation using strings.
+
+Because Cypress ignores unknown CLI parameters, you need to pass `fgrep` and `grep` arguments as environment variables, for example by using `--env` CLI argument.
+
+## Examples
+
+### fgrep to pick specs
+
+```bash
+# runs only tests from specs with "foo" in their filename
+$ npm run cypress:open -- --env fgrep=foo
+```
+
+Running all tests shows that all tests are skipped, except for tests in file [cypress/integration/foo.js](cypress/integration/foo.js)
+
+![fgrep foo](images/fgrep-foo.png)
+
+### grep to pick tests
+
+```bash
+# runs in headless mode, skipping all but one matching test
```

Comment:
I would prefer to see the example pulling out more than 1 matching test, since this is more likely their usecase. Also, It'd be better to have a more realistic example than 'runs fine', like grep for 'admin' where it runs all of the tests that match testing under 'admin' roles of your application.

Reply:
done